setup/src/Magento/Setup/Validator/IpValidator.php

```diff
@@ -66,14 +66,32 @@ public function validateIps(array $ips, $noneAllowed)
      */
     private function filterIps(array $ips)
     {
-        foreach ($ips as $ip) {
-            if (filter_var($ip, FILTER_VALIDATE_IP)) {
-                $this->validIps[] = $ip;
-            } elseif ($ip == 'none') {
-                $this->none[] = $ip;
-            } else {
-                $this->invalidIps[] = $ip;
+        foreach ($ips as $range) {
+            if ($range === 'none') {
+                $this->none[] = $range;
+                continue;
             }
+
+            $subnetMask = 32;
+            $ip = $range;
+            if (strpos($range, '/') !== false) {
+                [$ip, $subnetMask] = explode('/', $range);
+            }
+
+            $ipValidator = new \Laminas\Validator\Ip();
+            if (!$ipValidator->isValid($ip)) {
+                $this->invalidIps[] = $range;
+                continue;
+            }
+
+            $ipv4Validator = new \Laminas\Validator\Ip(['allowipv6' => false]);
```

Comment:
Please, use import with an alias instead of FQN here as well

Reply:
done